src/Apps/Artist/Components/NavigationTabs.tsx

```diff
@@ -62,9 +64,10 @@ export class NavigationTabs extends React.Component<Props> {
                   <RouteTabs
                     // FIXME: Don't use negative margins, adjust container padding responsively
                     mx={xs ? -4 : 0}
-                    pl={xs ? 4 : 0}
                     size={xs ? "xs" : null}
                   >
+                    {/* Add spacer on mobile */}
+                    {xs && <span style={{ paddingLeft: 20 }} />}
```

Comment:
Quickfix for an issue I noticed in staging today -- i think it was due to a padding component the purchase team introduced a little while back but this wasn't updated. (need to revist this later) 

<img width="300" alt="screen shot 2018-10-11 at 5 15 29 pm" src="https://user-images.githubusercontent.com/236943/46840769-87ed1a80-cd79-11e8-872c-18c279cb8c4d.png">